.github/ISSUE_TEMPLATE/new_metapackage.yml

```diff
@@ -51,7 +51,7 @@ body:
       description: Which category should this tool be installed to?
       options:
         - Active Directory
-        - Android
+        - Java/Android
```

Comment:
nit: `Android/Java`

Reply:
why?

Reply:
I prefer Android/Java but yeah main nit is the alphabetical order here

Reply:
> I prefer Android/Java

why?

Reply:
not a big deal, so feel free to go ahead and merge

Reply:
I was just curious because I had it the other way around and then I changed it because I though Java is more general. I have sorted it alphabetically now..